tokio/tests/io_util_empty.rs

```diff
@@ -30,3 +31,16 @@ async fn empty_buf_reads_are_cooperative() {
         _ = tokio::task::yield_now() => {}
     }
 }
+
+#[tokio::test]
+async fn empty_seek() {
```

Comment:
Can we add more tests like the `empty_seeks` in  https://github.com/rust-lang/rust/commit/f1cd17961ccaac4bfaeeab81969cf36c56eec4a5?